### ISSUE TITLE
Link editing is terrible, easy to make mistakes

### ISSUE THREAD

**Issue by quantumleap33** [ThemeFuse/Brizy#799]
When I try to add or remove a link from some text, there is an issue sometimes when for some reason all text that is part of the paragraph block gets applied as a link! Not only that but the undo doesn't work and combined with the fact that Brizy actually saves whatever you do, there is no way to get back to the original text. Even if I don't hit Save, go back to WordPress and come back editing with Brizy the changes are still there! It's little things like that that make Brizy for me completely unusable.

**Comment by Dabamash** [comment]
Yes, agree with all of these. One of the most frustrating things about the builder at the moment.

**Comment by Infusednl** [comment]
+1, I also experience this issue and hate it. However they are aware of this bug so hopefully they will fix it soon.

**Comment by Ravisi** [comment]
I have the same problem, hope will be fixed soon, it's frustrating.

**Comment by willhousecreative** [comment]
Agreed. I even tried to go "under the hood" and edit the page using the Wordpress editor, manually adding links in the HTML. When I update and check out the page it's replaced with the theme template. The only thing to do is to go back to the Brizy editor where none of my edits to the HTML are saved. It just goes right back to the page with the link problems. So frustrating!

**Comment by bagrinsergiu** [comment]
Hi All, 

Link Editing + Typography Imput Fields are planned for 1.0.45. Thanks for suggestions.

Sorry for the delay
Sergiu

**Comment by bagrinsergiu** [comment]
Hi again, 

This issue was delayed to 1.0.46 

Sergiu
